architecture/users/readme.md

```diff
@@ -1,40 +1,34 @@
 # User service
 
-Микросервис предоставляет пользователям (другим микросервисам) данные о пользователях курса (студентах). 
+The service provides the data about course students for users (other services).
 
-Данные о пользователях хранятся в CRM, работа с которой осуществляется через API (т.о. на данный момент слой данных - это CRM). Микросервис предоставляет пользователям ограниченный RESTful API (без возможности удаления записей).
+Student's data is stored in the PostgreSQL database. The records in the database are synchronized daily with the source CRM on a schedule.
```

Comment:
The records in the database are synchronized ~daily~ with the source CRM on ~a~ schedule **once per day**.